variables.tf

```diff
@@ -0,0 +1,48 @@
+variable "environment" {
+  description = "An environment qualifier for the resources this module creates, e.g. staging, or prod."
+  type        = string
+}
+
+variable "gcp_org_id" {
+  description = "The Organization ID of your Google Cloud Organization"
+}
+
+variable "identity_pool_project_id" {
+  description = "The Project ID of the Google Cloud Project where the Workload Identity Federation resources will be created in."
+  type        = string
+}
+
+variable "enable_google_group_management" {
+  description = "A boolean indicating whether to enable the Admin SDK API to allow the Sym Integration to manage Google Group membership."
+  type        = bool
+  default     = false
+}
+
+variable "google_workspace_customer_id" {
+  description = "The Google Workspace Customer ID. Required if managing Google Group memberships."
+  type        = string
+  default     = null
+}
+
+output "validate_google_workspace_customer_id" {
+  # A workaround to validate that `google_workspace_customer_id` is not empty, if managing Google Groups.
+  # See: https://github.com/hashicorp/terraform/issues/25609#issuecomment-1472119672
+
+  value = null
+  precondition {
```

Comment:
This precondition block is what forces our minimum Terraform version to be `>= 1.2`. 

Currently, the latest version of Terraform is 1.6.5. Are we comfortable with saying that to use the `gcp_connector` you must have a version >= 1.2?